modules/server/certbot/certbot.timer.tmpl

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Runs Certbot periodically
+
+[Timer]
+OnCalendar=${frequency}
+Unit=certbot.service
```

Comment:
Implicit - see: https://www.freedesktop.org/software/systemd/man/systemd.timer.html#Unit=

Reply:
Done.
I randomly named this file *.timer, without searching in the docs, thanks, nice feature ;)